include/fiction/algorithms/physical_design/ortho_ordering_network.hpp

```diff
@@ -0,0 +1,772 @@
+//
+// Created by benjamin on 11.04.23.
+//
+
+#ifndef FICTION_ORTHO_ORDERING_NETWORK_HPP
+#define FICTION_ORTHO_ORDERING_NETWORK_HPP
+
+#include "fiction/algorithms/network_transformation/inverter_substitution.hpp"
+#include "fiction/algorithms/physical_design/orthogonal.hpp"
+#include "fiction/networks/views/input_ordering_view.hpp"
+
+#include <algorithm>
+#include <cstdint>
+#include <cstdlib>
+#include <vector>
+
+namespace fiction
+{
+
+namespace detail
+{
+/**
+ * Paints a node and all incoming edges
+ */
```

Comment:
This docstring doesn't document any of the parameters and template parameters.

Same goes for most other functions in this file. Can you fix this, please?